### PR TITLE
Renewal item database update

### PR DESCRIPTION
**Pull request by crazystorm2017**
* **Addressed Issue(s)**: 
#4111

* **Server Mode**: Renewal

* **Description of Pull Request**: 

  - Added Calf Weapons
  - Added King Schmidt's Accessories
  - Added King Schmidt's Equipment/Combos
  - Added New Spears 
  - Added Missing Costumes
  - Remove Costume Refineability

### CODE REVIEW

File: db/re/item_db.txt [hunk 1]
```diff
@@ -11996,6 +11997,10 @@
...
+28240,Calf_Kingcobra,Calf Kingcobra,5,0,0,900,230,,9,2,0x40000000,63,2,34,4,150,1,18,{ .@r = getrefine(); bonus bAspdRate,10 + (.@r > 6 ? 5 : 0); bonus bLongAtkRate,(.@r/2)*5 + (.@r > 11 ? 5 : 0); if(.@r > 8){ bonus bCritAtkRate,15; } },{},{}
+28241,Calf_Diamondback,Calf Diamondback,5,0,,3000,220,,9,2,0x40000000,63,2,34,4,150,1,19,{ .@r = getrefine(); bonus bDex,2; bonus bAtkRate,2; bonus bLongAtkRate,(.@r/2)*5 + (.@r > 11 ? 5 : 0); if(.@r > 6){ bonus2 bSkillAtk,"RL_R_TRIP",20; } if(.@r > 8){ bonus2 bSkillUseSPrate,"RL_R_TRIP",-5; } },{},{}
+28242,Calf_Anaconda,Calf Anaconda,5,10,,1400,305,,9,2,0x40000000,63,2,34,4,150,1,20,{ .@r = getrefine(); bonus bSplashRange,1; bonus bLongAtkRate,(.@r/2)*5 + (.@r > 8 ? 5 : 0); if(.@r > 6){ bonus2 bSkillAtk,"RL_BANISHING_BUSTER",20; } if(.@r > 11){ bonus2 bSkillUseSPrate,"RL_BANISHING_BUSTER",-15; } },{},{}
+28243,Calf_Python,Calf Python,5,10,0,2000,360,,9,2,0x40000000,63,2,34,4,150,1,21,{ bonus bDex,2; bonus bAtkRate,2; bonus bLongAtkRate,(.@r/2)*5; bonus2 bSkillAtk,"RL_D_TAIL",(.@r > 11 ? 15 : (.@r > 8 ? 15 : 0)); if(.@r > 6){ bonus bAspdRate,5; } },{},{}
```
Comment (Atemo) [inline db/re/item_db.txt]: The damage of Dragon Tail should be increased by 30% (15+15) at +12 or above.

File: db/re/item_db.txt [hunk 2]
```diff
@@ -12611,26 +12617,151 @@
...
+31607,C_Cat_Ear_Witch_Hat,C Cat Ear Witch Hat,4,0,,,,0,,0,0xFFFFFFFF,63,2,1024,,1,0,1808,{},{},{}
+31608,C_Magicstone_of_Grace,C Magic Stone of Grace,4,0,,,,0,,0,0xFFFFFFFF,63,2,7168,,1,0,1420,{},{},{}
+31609,C_Noble_Mask,C Noble Mask,,0,,,,0,,0,0xFFFFFFFF,63,2,2048,,1,,1409,{},{},{}
+31610,C_Yellow_Wizardry_Hat,C Yellow Wizardry Hat,4,0,,,,0,,0,0xFFFFFFFF,63,2,1024,,1,0,286,{},{},{}
+31611,C_Snake_Lord_Stole,C Snake Lord Stole,4,0,,,,0,,0,0xFFFFFFFF,63,2,4096,,1,0,1810,{},{},{}
+31612,C_Large_Sorcerer_Crown,C Large Sorcerer Crown,4,0,,,,0,,0,0xFFFFFFFF,63,2,1024,,1,0,1811,{},{},{}
 //
 32005,Illusion_Pole_Axe,Illusion Pole Axe,5,0,,1500,190,,3,2,0x00004082,63,2,2,4,120,1,4,{ .@r = getrefine(); bonus bStr,3; bonus bInt,2; bonus bDex,2; if (.@r >= 13) { .@val = -2000; } else if (.@r >= 9) { .@val = -1000; } bonus2 bSkillCooldown,"RK_HUNDREDSPEAR",.@val; },{},{}
+32013,Metal_Stick,Metal Stick,5,0,0,2000,180,,3,3,0x00004000,63,2,2,4,150,1,4,{ .@r = getrefine(); bonus2 bSkillAtk,"LG_CANNONSPEAR",10; bonus2 bSkillAtk,"LG_OVERBRAND",(.@r > 8 ? 15 : (.@r > 6 ? 10 : 0)); if(.@r >= 12){ bonus bDelayrate,-15; } },{},{}
```
Comment (Atemo) [inline db/re/item_db.txt]: - The damage of `LG_OVERBRAND` should be increased by 25% (10+15%) at +9 or above.
- `Jobs: 3rd classes swordsman` means the equippable jobs should be `0x00004080` and the equippable upper-types should be `56`.
```
	Knight         (2^07): 0x00000080
	Crusader       (2^14): 0x00004000
```
```
	 8: Third classes (no Transcedent-Third or Third-Baby classes)
	16: Transcedent-Third classes
	32: Third-Baby classes
```

File: db/re/item_db.txt [hunk 3]
```diff
@@ -12611,26 +12617,151 @@
...
+31574,C_Cyber_Income,C Cyber Income,4,0,,0,,0,,0,0xFFFFFFFF,63,2,2048,,1,0,1790,{},{},{}
+31575,C_Electro_Two_Sides_Up,C Electro Two Sides Up,4,0,,0,,0,,0,0xFFFFFFFF,63,2,4096,,1,0,1791,{},{},{}
+31576,C_Moon_Rabbit_Hat,C Moon Rabbit Hat,4,0,,0,,0,,0,0xFFFFFFFF,63,2,1024,,1,0,455,{},{},{}
+31577,C_Roast_Memory_Upper,C Roast Memory Upper,4,0,,0,,0,,0,0xFFFFFFFF,63,2,1024,,1,0,1576,{},{},{}
+31578,C_Alchemist_Mask_ML,C Alchemist Mask,4,0,,0,,0,,0,0xFFFFFFFF,63,2,6144,,1,0,1497,{},{},{}
+31579,C_Love_Games,C I LOVE GAMES,4,0,,0,,0,,0,0xFFFFFFFF,63,2,1024,,1,0,1306,{},{},{}
+31580,C_Autumn_Flavor,C Autumn Flavor,4,0,,0,,0,,0,0xFFFFFFFF,63,2,2048,,1,0,1792,{},{},{}
+31581,C_Jack_Upper,C Jack Upper,4,0,,,,0,,0,0xFFFFFFFF,63,2,1024,,1,0,1415,{},{},{}
+31582,C_Jirant_Circlet,C Jirant Circlet,4,0,,,,0,,0,0xFFFFFFFF,63,2,1024,,1,0,1644,{},{},{}
+31583,C_Will_O_Wisp_Lower,C Will O Wisp Lower,4,0,,,,0,,0,0xFFFFFFFF,63,2,4096,,1,0,880,{},{ sc_start SC_STRANGELIGHTS,INFINITE_TICK,0; },{}
```
Comment (Atemo) [inline db/re/item_db.txt]: The effect should be removed when the costume is unequipped.

File: db/re/item_db.txt [hunk 4]
```diff
@@ -12611,26 +12617,151 @@
...
+31610,C_Yellow_Wizardry_Hat,C Yellow Wizardry Hat,4,0,,,,0,,0,0xFFFFFFFF,63,2,1024,,1,0,286,{},{},{}
+31611,C_Snake_Lord_Stole,C Snake Lord Stole,4,0,,,,0,,0,0xFFFFFFFF,63,2,4096,,1,0,1810,{},{},{}
+31612,C_Large_Sorcerer_Crown,C Large Sorcerer Crown,4,0,,,,0,,0,0xFFFFFFFF,63,2,1024,,1,0,1811,{},{},{}
 //
 32005,Illusion_Pole_Axe,Illusion Pole Axe,5,0,,1500,190,,3,2,0x00004082,63,2,2,4,120,1,4,{ .@r = getrefine(); bonus bStr,3; bonus bInt,2; bonus bDex,2; if (.@r >= 13) { .@val = -2000; } else if (.@r >= 9) { .@val = -1000; } bonus2 bSkillCooldown,"RK_HUNDREDSPEAR",.@val; },{},{}
+32013,Metal_Stick,Metal Stick,5,0,0,2000,180,,3,3,0x00004000,63,2,2,4,150,1,4,{ .@r = getrefine(); bonus2 bSkillAtk,"LG_CANNONSPEAR",10; bonus2 bSkillAtk,"LG_OVERBRAND",(.@r > 8 ? 15 : (.@r > 6 ? 10 : 0)); if(.@r >= 12){ bonus bDelayrate,-15; } },{},{}
+32014,Brute_Spear,Brute Spear,5,0,0,,150,,3,0,0x00004080,63,2,2,4,100,1,4,{ .@r = getrefine(); bonus bUnbreakableWeapon; bonus2 bIgnoreDefRaceRate,RC_All,5*.@r; bonus2 bAddRace,RC_Player,80+(.@r > 8 ? 20 : (.@r > 7 ? 35 : 0)); if(.@r > 9){ bonus bMaxHPrate,15; bonus bMaxSPrate,15; } },{},{}
```
Comment (Atemo) [inline db/re/item_db.txt]: - The equippable upper-types should be `56` (`Jobs: 3rd Job Swordman class`)
```
	 8: Third classes (no Transcedent-Third or Third-Baby classes)
	16: Transcedent-Third classes
	32: Third-Baby classes
```

- The damage against player at +9 or above should be +(35+20)%.

File: db/re/item_db.txt [hunk 5]
```diff
@@ -8823,6 +8823,8 @@
 //===================================================================
 // More Maces
 //===================================================================
+15388,King_Schmidt_Suit,King Schmidt's Suit,4,0,0,900,,150,,1,0xFFFFFFFF,63,2,16,,130,1,,{ .@r = getrefine(); bonus2 bAddSize,Size_Medium,10; bonus2 bAddSize,Size_Large,10; if(.@r > 6){ bonus2 bAddEle,Ele_Fire,10; bonus2 bAddEle,Ele_Dark,10; bonus2 bMagicAddEle,Ele_Fire,10; bonus2 bMagicAddEle,Ele_Dark,10; } if(.@r > 8){ bonus2 bSubRace,RC_Dragon,15; bonus2 bSubRace,RC_Formless,15; } if(.@r > 10){ bonus2 bSubSize,Size_Medium,10; bonus2 bSubSize,Size_Large,10; } },{},{}
```
Comment (Atemo) [inline db/re/item_db.txt]: The bonus `bMagicAddSize` is missing vs `Size_Medium` and `Size_Large`.

File: db/re/item_combo_db.txt [hunk 6]
```diff
@@ -850,4 +851,10 @@
 28907:2998,{ bonus bSpeedRate,25; }
 28908:2998,{ bonus bDelayrate,-10; }
 28909:2998,{ bonus bUseSPrate,-10; bonus bMaxSPrate,10; }
+32228:15388:15389,{ bonus bAtk,25; bonus bHit,10; }
```
Comment (Atemo) [inline db/re/item_combo_db.txt]: Please use `bBaseAtk` instead of `bAtk` (unofficial).

File: db/re/item_combo_db.txt [hunk 7]
```diff
@@ -850,4 +851,10 @@
...
+32229:15388:15389,{ bonus bMatk,25; bonus bDelayrate,-8; }
```
Comment (Atemo) [inline db/re/item_combo_db.txt]: The combo should reduce the variable casting time instead of the skill delay.

**Comment by aleos89** [comment]
Do you mind syncing this so it can be merged to master? Thanks!

**Comment by Atemo** [comment]
Thanks!

**Pull request merged**